bvm/ballerina-runtime/src/test/java/io/ballerina/runtime/test/config/ConfigTest.java

```diff
@@ -122,13 +120,6 @@ public void testTomlConfigProviderWithSimpleTypes(VariableKey key, Class<?> expe
                 {new VariableKey(module, "decimalVar", PredefinedTypes.TYPE_DECIMAL, true), DecimalValue.class,
                         new DecimalValue("876.54"),
                         new CliProvider(ROOT_MODULE, "-CmyOrg.test_module.decimalVar=876.54")},
-                // Xml value given only with cli
```

Comment:
Why are we removing this?

Reply:
After discussing with @HindujaB it was suggested that it's better to remove the XML test from this to prevent the test from getting more complex and also because a similar test is already there in integration tests.

Reply:
Yes. We have a separate integration test for checking this behavior (configurable xml values through CLI args).
https://github.com/ballerina-platform/ballerina-lang/blob/ddc7d1f1bc769689f6a5793cb9d677c7ad348305/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java#L84

And if we add the Typechecker check for this case, it will add another dependency to the unit test which IMO will make it more complex. 
Therefore, we can remove this test.